src/_atomic_data.cpp

```diff
@@ -17,6 +18,26 @@ void pyne::_load_atomic_mass_map_memory() {
   } else { 
     _insert_abund_map();
   }
+
+  // calculate the atomic_masses of the elements
+  std::map<int,double> :: iterator it;
+
+  for ( int i = 0 ; i < 92 ; i++ ) {
```

Comment:
From a style guide perspective, it seems like there is too much whitespace in this line. (Maybe drop the spaces just inside the parentheses?) https://google.github.io/styleguide/cppguide.html#Horizontal_Whitespace .

Also, this loop would be a little more clear (and actually would avoid declaring `int atomic_number = i + 1`) if the for loop explicitly looped over atomic number. 

So, this is just preference for brevity, but I would suggest something along the lines of:

``` 
for (int z=1; z<=92; z++){
     double element_atomic_weight = 0.0;
     ...
           if(pyne::nucname::znum(it->first) == z ) {
     ...
     
```